src/components/Sing/SequencerPitch.vue

```diff
@@ -464,6 +464,6 @@ onUnmountedOrDeactivated(() => {
   z-index: 0;
   pointer-events: none;
 
-  contain: content;
+  contain: strict;
```

Comment:
意図がわからなくなりそうなのでコメント付けときますか
```suggestion
  contain: strict; // canvasのサイズが変わるのを無視する
```